sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,3 +1,9 @@
+# 2019-05-20 1.0.2
+
+- The build and packaging pipeline is updated to use npm version 6.9.0 which has the fix to allow nested
```

Comment:
Pipeline seems like excess detail. Maybe:

> Added missing package.json file to the npm package to fix issues bundling with webpack and other bundlers.

Reply:
You are writing the changelog from now on :)